NativeApp/include/AppBase.hpp

```diff
@@ -156,6 +169,12 @@ class AppBase
     {
         return false;
     }
+
+    /// Returns default hotkeys handling flags
+    virtual HOT_KEY_FLAGS GetHotKeyFlags() const
+    {
+        return HOT_KEY_FLAG_ALLOW_EXIT_ON_ESC | HOT_KEY_FLAG_ALLOW_FULL_SCREEN_SWITCH;
```

Comment:
Please return the `HOT_KEY_FLAG_ALL` here. If new enum values are add, it is almost guaranteed that this function will not be updated.